package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "myfooter",
+	"version": "0.0.1",
+	"description": "A pretty footer",
+	"main": "lib/index.js",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"build": "rm -rf ./lib && tsc",
+		"prepare": "yarn build"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/suzynakayama/2020-04-17-personal-npm-package.git"
+	},
+	"author": "Suzy Nakayama",
+	"license": "MIT",
+	"bugs": {
+		"url": "https://github.com/suzynakayama/2020-04-17-personal-npm-package/issues"
+	},
+	"homepage": "https://github.com/suzynakayama/2020-04-17-personal-npm-package#readme",
+	"devDependencies": {
```

Comment:
React is a dependency because it is included in the final build so it should go in dependencies I think, don't quote me on this